src/Components/Web.JS/src/Platform/WebAssemblyResourceLoader.ts

```diff
@@ -39,12 +39,23 @@ export class WebAssemblyResourceLoader {
       return { name, url, response };
   }
 
-  logToConsole() {
+  logStatistics() {
+    // Used by test \ benchmarking infrastructure
+    const onDownloadStats = window['Blazor']._internal.downloadStats;
```

Comment:
This seemed "easy" to do, but if there's a better way to approach this, let me know.